src/lifx.d.ts

```diff
@@ -1,367 +1,328 @@
-export class Client {
+import {RemoteInfo} from 'dgram';
+import {EventEmitter} from 'eventemitter3';
+import {AddressInfo} from 'net';
```

Comment:
If we're referencing types from other packages, I believe we should have them as dev dependencies in package.json also.

Reply:
dgram and net are both core node modules, eventemitter3, sure thing.